qiskit_metal/qlibrary/resonator/readoutres_fc.py

```diff
@@ -0,0 +1,189 @@
+import numpy as np
+import sys
+
+from qiskit_metal import draw, Dict
+from qiskit_metal.qlibrary.core import BaseQubit
+
+class ReadoutRes_fc(BaseQubit):
+    '''
+    The base ReadoutRes_fc class
+
+    Inherits BaseQubit class
+
+    Readout resonator for the flipchip dev. Written for the flipchip tutorial.
```

Comment:
Will want to expand the documentation here to mimic what is seen with other components.

Also thinking maybe not wanting BaseQubit as the parent class, as that makes renderers expect there to be a JosephsonJunction to be present in the component.

Reply:
1) Thanks for the feedback thomas, sorry for the lack of explanatory docstrings as I thought that objects like this are not something you would like to keep in the long term since it lacks some generality that one would typically want to find when rendering a transmission line resonator. I can certainly improve the documentation in upcoming updates.
Actually, I wanted to draw the transmission line part of the resonator using QRoute and the circular head shape using the shapely function directly, then combine it within one class object. But at the time, I wasn't super clear how things would be grouped and named in ansys, and without taking the time to test through things and browse through QRoute implementations, I reverted to a more simpler way, which was to draw everything using the shapely function directly, something I understood a bit better. It is likely though that I'll phase out this way of drawing cpw resonators in the future in favor of using QRoute.

2) I used BaseQubit as the parent class and set the _qgeometry_table_junction in the component_metadata dictionary as False, which I thought should let the renderers know that there are no JosephsonJunction in the object? I wasn't exactly sure why I used BaseQubit as the parent class. It is very likely that I was thinking of rendering a tunable cpw resonator (with a jj). In any case, I can instead use QComponent as the parent class, and I think there should be no problem. Will change this in upcoming updates.

3) I wasn't exactly sure where I should place this resonator object in the qlibrary folder. For the moment I've intentionally created a specific folder for this, but if you feel that it should belong somewhere else, please let me know. The same for the tutorial notebook, I wasn't sure which folder it should belong to.